src/components/scenes/CoinRankingScene.tsx

```diff
@@ -197,51 +198,56 @@ const CoinRankingComponent = (props: Props) => {
+            <View style={styles.searchTextInputContainer}>
+              <OutlinedTextInput
+                returnKeyType="search"
+                label={lstrings.search_assets}
+                onChangeText={handleOnChangeText}
+                value={searchText ?? ''}
+                onFocus={handleTextFieldFocus}
+                onSubmitEditing={handleSubmit}
+                ref={textInput}
+                marginRem={0}
+                searchIcon
+              />
+            </View>
+            {searching && (
+              <TouchableOpacity onPress={handleSearchDone} style={styles.searchDoneButton}>
+                <EdgeText style={styles.tappableHeaderText}>{lstrings.string_done_cap}</EdgeText>
+              </TouchableOpacity>
+            )}
+          </View>
 
-      <View style={styles.container}>
-        <View style={styles.rankView}>
-          <EdgeText style={styles.rankText}>{lstrings.coin_rank_rank}</EdgeText>
-        </View>
-        <TouchableOpacity style={styles.assetView} onPress={handlePriceSubText}>
-          <EdgeText style={styles.assetText}>{assetSubTextString}</EdgeText>
-        </TouchableOpacity>
-        <TouchableOpacity style={styles.percentChangeView} onPress={handlePercentChange}>
-          <EdgeText style={styles.percentChangeText}>{timeFrameString}</EdgeText>
-        </TouchableOpacity>
-        <View style={styles.priceView}>
-          <EdgeText style={styles.priceText}>{lstrings.coin_rank_price}</EdgeText>
-        </View>
-      </View>
-      <DividerLine marginRem={[0, 0, 0, 1]} />
-      <FlashList
-        estimatedItemSize={theme.rem(3.75)}
-        data={listdata}
-        extraData={extraData}
-        renderItem={renderItem}
-        onEndReachedThreshold={1}
-        onEndReached={handleEndReached}
-      />
+          <View style={styles.container}>
+            <View style={styles.rankView}>
+              <EdgeText style={styles.rankText}>{lstrings.coin_rank_rank}</EdgeText>
+            </View>
+            <TouchableOpacity style={styles.assetView} onPress={handlePriceSubText}>
+              <EdgeText style={styles.assetText}>{assetSubTextString}</EdgeText>
+            </TouchableOpacity>
+            <TouchableOpacity style={styles.percentChangeView} onPress={handlePercentChange}>
+              <EdgeText style={styles.percentChangeText}>{timeFrameString}</EdgeText>
+            </TouchableOpacity>
+            <View style={styles.priceView}>
+              <EdgeText style={styles.priceText}>{lstrings.coin_rank_price}</EdgeText>
+            </View>
+          </View>
+          <DividerLine marginRem={[0, 0, 0, 1]} />
+          <FlashList
+            estimatedItemSize={theme.rem(3.75)}
+            data={listdata}
+            extraData={extraData}
+            renderItem={renderItem}
+            onEndReachedThreshold={1}
+            onEndReached={handleEndReached}
+            contentContainerStyle={{ paddingBottom: gap.bottom }}
```

Comment:
`style={ marginBottom: -gap.bottom }`

Reply:
No this is right, it should be positive. The notificationHeight is meant as an
contentContainerStyle paddingBottom (overscroll) - using it as a style padding causes a visual cutoff